src/ucs/sys/sock.c

```diff
@@ -391,6 +391,13 @@ ucs_status_t ucs_socket_server_init(const struct sockaddr *saddr, socklen_t sock
         if (status != UCS_OK) {
             goto err_close_socket;
         }
+#ifdef SO_REUSEPORT
```

Comment:
IMO better check it in configure (theoretically, it could be enum/constant and not a macro)
and use `#ifdef HAVE_DECL_SO_REUSEPORT`